aws-lc-sys/builder/main.rs

```diff
@@ -77,21 +77,28 @@ fn option_env<N: AsRef<str>>(name: N) -> Option<String> {
 fn env_var_to_bool(name: &str) -> Option<bool> {
     let build_type_result = option_env(name);
     if let Some(env_var_value) = build_type_result {
-        eprintln!("{name}={env_var_value}");
-        // If the environment variable is set, we ignore every other factor.
+        eprintln!("Evaluating: {name}='{env_var_value}'");
+
         let env_var_value = env_var_value.to_lowercase();
         if env_var_value.starts_with('0')
             || env_var_value.starts_with('n')
             || env_var_value.starts_with("off")
+            || env_var_value.starts_with('f')
         {
-            Some(false)
-        } else {
-            // Otherwise, if the variable is set, assume true
-            Some(true)
+            eprintln!("Parsed: {name}=false");
+            return Some(false);
         }
-    } else {
-        None
+        if env_var_value.starts_with(|c: char| c.is_ascii_digit())
+            || env_var_value.starts_with('y')
+            || env_var_value.starts_with("on")
+            || env_var_value.starts_with('t')
+        {
```

Comment:
Should this handle `1` as true like you do for false up above?

Reply:
I thought about only accepting "1" for `true`, but then thought it's also reasonable to accept "2" as `true`. Which led me to thinking that any non-zero digit could reasonably be considered `true` if being parsed into a `boolean` value.